samples-generator/bin/3d-tiles-samples-generator.js

```diff
@@ -956,11 +1014,14 @@ function savePointCloudTileset(tilesetName, tileOptions, tilesetOptions) {
     var result = createPointCloudTile(tileOptions);
     var pnts = result.pnts;
     var batchTableJson = result.batchTableJson;
+    var extensionsUsed = result.extensionsUsed;
```

Comment:
I added an `Extensions` class in https://github.com/AnalyticalGraphicsInc/3d-tiles-tools/pull/149/files, do you want to use that?